packages/lesswrong/lib/collections/votes/schema.ts

```diff
@@ -179,6 +179,14 @@ const schema: SchemaType<DbVote> = {
     type: Boolean,
     canRead: ['guests'],
     ...schemaDefaultValue(false)
+  },
+
+  // Whether to notify users of the karma changes from this vote. This is set to false for votes that are
+  // nullified by mod actions
+  notify: {
+    type: Boolean,
+    canRead: ['guests'],
+    ...schemaDefaultValue(true)
```

Comment:
Can you mark this as explicitly `nullable: false`?  While https://github.com/ForumMagnum/ForumMagnum/pull/8051 and its sibling PR are still in the works, I'd like to have all new fields explicitly specify nullability.  (And in this case `nullable: true` would leave previous votes with the wrong implicit value.)

Reply:
I forget whether add field will update the database to set this default value on all previous votes, but if it don't then it's the right implicit value.

Reply:
Oh, hm, it probably does, yeah.  (Still want to avoid building up additional debt with mismatched generated db types & actual db schema.)